srcpkgs/ol/template

```diff
@@ -10,13 +10,26 @@ license="LGPL-3.0-or-later, MIT"
 homepage="https://yuriy-chumak.github.io/ol/"
 changelog="https://raw.githubusercontent.com/yuriy-chumak/ol/master/doc/CHANGELOG.md"
 distfiles="https://github.com/yuriy-chumak/ol/archive/${version}.tar.gz"
-checksum=5a7b8ff5eeba2db63ba8d409c6ebc3af5f9cb21d14a2ec1a279d8137298360ef
+checksum=c0d0beadbd549e453a93dba350ed18b2df070f783c1803cb18ebcc5518ed6e8e
 
 case "$XBPS_TARGET_MACHINE" in
-	i686*|x86_64*) makedepends+=" gcc-multilib";;
+	i686*)   export HAS_64CDEFS=0;;
+	x86_64*) export HAS_32CDEFS=0;;
 esac
```

Comment:
If I'm understanding this right, what you actually want is
```bash
if [[ "$XBPS_TARGET_WORDSIZE" -eq 32 ]]; then
	export HAS_64CDEFS=0
else
	export HAS_32CDEFS=0
fi
```

Reply:
Yeah, thank you. That's better.
(I changed to `make_check_args` because that's for `make check`).
I can see the regression tests are running. Fine!